tests/test_data_model.py

```diff
@@ -71,13 +71,30 @@ def test_lstm_simple(self):
         # Create from model
         LSTMStateTransitionModel(m.model, output_keys = ['x'])
 
+        # Test pickling model m
+        import pickle
+        pickled_m = pickle.dumps(m)
+        m2 = pickle.loads(pickled_m)
+        self.assertIsInstance(m2, LSTMStateTransitionModel)
+        self.assertIsInstance(m2, DataModel)
+        self.assertListEqual(m2.outputs, ['x'])
+
         # More tests in examples.lstm_model
 
     def test_dmd_simple(self):
-        self._test_simple_case(DMDModel, max_error=4)
+        self._test_simple_case(DMDModel, max_error=6)
 
         # Without velocity, DMD doesn't perform well
-        self._test_simple_case(DMDModel, WITH_STATES = False, max_error=100)
+        m = self._test_simple_case(DMDModel, WITH_STATES = False, max_error=100)
+
+        # Test pickling model m
+        import pickle
+        pickled_m = pickle.dumps(m)
+        m2 = pickle.loads(pickled_m)
+        self.assertIsInstance(m2, DMDModel)
+        self.assertIsInstance(m2, DataModel)
+        self.assertListEqual(m2.outputs, ['x'])
+
```

Comment:
Test passes, but the following warning is generated:

```
UserWarning: State value 'x' is duplicated in inputs, outputs, or events; duplicate has been removed.
```

Nothing stands out immediately that causes this (likely can be ignored?)

Reply:
That warning's expected. DMD is trying to be smart and eliminate duplicate keys.